src/resolvers.js

```diff
@@ -28,8 +34,10 @@ export class Configuration {
    *
    * @return {Homefront}
    */
-  get(container: Container): Homefront {
-    return container.get(Config).use(this._namespace);
+  get(container: Container): {}|Homefront {
+    return (this._asHomefront
+            ? container.get(Config).use(this._namespace)
+            : container.get(Config).fetch(this._namespace));
```

Comment:
@RWOverdijk 
i think inject per default as object is the better choice
or should that be seperate resolvers?